unlock-app/src/pages/log.js

```diff
@@ -0,0 +1,116 @@
+import React from 'react'
+import { connect } from 'react-redux'
+import Head from 'next/head'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import UnlockPropTypes from '../propTypes'
+import Layout from '../components/interface/Layout'
+import CreatorAccount from '../components/creator/CreatorAccount'
+import BrowserOnly from '../components/helpers/BrowserOnly'
+import GlobalErrorConsumer from '../components/interface/GlobalErrorConsumer'
+import GlobalErrorProvider from '../utils/GlobalErrorProvider'
+import { pageTitle } from '../constants'
+import configure from '../config'
+
+const { chainExplorerUrlBuilders } = configure()
+
+export const humanize = type => {
+  if (!type) return ''
+  let parts = type.split('_')
+  return parts.map(part => part[0] + part.slice(1).toLowerCase()).join(' ')
+}
+
+export const Log = ({ account, network, transactionFeed }) => {
+  return (
+    <GlobalErrorProvider>
+      <GlobalErrorConsumer>
+        <Layout title="Creator Log">
+          <Head>
+            <title>{pageTitle('Log')}</title>
+          </Head>
+          <BrowserOnly>
+            <CreatorAccount network={network} account={account} />
+            <Body>
+              <LogHeader>Block Number</LogHeader>
+              <LogHeader>Lock Name/Address</LogHeader>
+              <LogHeader>Type</LogHeader>
+              {transactionFeed.length > 0 &&
+                transactionFeed.map(tx => (
+                  <React.Fragment key={tx.hash}>
+                    <LogElement>{tx.blockNumber}</LogElement>
+                    <Address
+                      href={
+                        chainExplorerUrlBuilders.etherScan(tx.lock) || undefined
```

Comment:
Do this in the mapStateToProps, we should only have dumb data in the component (this will make converting to anything non-redux in the future easier)